components/Bounty/AwardBountyModal.tsx

```diff
@@ -610,7 +620,7 @@ function AwardBountyModal({
           rippleClass={styles.awardRipple}
           customLabelStyle={styles.labelStyle}
           onClick={awardBounty}
-          disabled={bountyAwardLoading}
+          disabled={bountyAwardLoading || threads?.length < 1}
```

Comment:
null check over falsey check 
```suggestion
          disabled={bountyAwardLoading ?? threads?.length < 1}
```